public/js/newsManagerController.js

```diff
@@ -0,0 +1,39 @@
+(function(exports){
+
+function NewsManagerController (storySummary, headlines, NewsManagerModel, NewsManagerView, ApiManager) {
+  this.storySummary = storySummary;
+  this.headlines = headlines;
+  this.model = NewsManagerModel;
+  this.view = NewsManagerView;
+  this.api = ApiManager;
+}
+
+NewsManagerController.prototype = {
+
+  createStories: function(JSON, self) {
+    self.model.saveNews(JSON);
+    self.view.createLinks(self.model.storyList);
+    self.setupHeadlines();
+  },
+
+  showSummary: function(id, newsSummary, self){
```

Comment:
We covered why you're passing in the controller in the group code review.  Definitely worth changing this code so the methods can be called with (eg) `controller.showSummary` and `self` isn't passed in and lines like `self.view.invisible('headlines');` can be `this.view.invisible('headlines');`.  Reading more about JavaScript context will help with how to do this.  Happy to talk it through if you get stuck!